themes/socialbase/templates/system/html.html.twig

```diff
@@ -59,5 +60,14 @@
     {% block svg_icons %}
       {{ svg_icons|raw }}
     {% endblock %}
+
+    {% if style == 'sky' %}
```

Comment:
We should be able to alter the `svg_icons` variable if needed. We should also consider whether it's correct for SocialBlue to have icons that are not used in SocialBase (cc @xinyumaGG) because SocialBase should be the "black and white" version of SocialBlue.